sync/class.jetpack-sync-actions.php

```diff
@@ -204,6 +199,22 @@ static function do_initial_sync( $new_version = null, $old_version = null ) {
 		self::do_full_sync( $initial_sync_config );
 	}
 
+	static function do_inital_sync_on_site_registration() {
```

Comment:
You're not using the action parameter here, that's even better. You don't need to do anything to the old action, you can just use it :)

Reply:
I am using the new action now.